### PR TITLE
Aruha 366 hila user journey

### PR DESCRIPTION
**Pull request by v-stepanov**
User journey test for Subscription API
@rcillo @antban @adyach @v1ctor please review

### CODE REVIEW

File: src/acceptance-test/java/org/zalando/nakadi/webservice/utils/TestStreamingClient.java [hunk 1]
```diff
@@ -28,6 +29,7 @@
     private final List<StreamBatch> batches;
     private InputStream inputStream;
     private String sessionId;
+    private Optional<String> token;
```
Comment (adyach) [inline src/acceptance-test/java/org/zalando/nakadi/webservice/utils/TestStreamingClient.java]: final

**Comment by adyach** [comment]
Failing for sandbox:
{"type":"http://httpstatus.es/422","title":"Unprocessable Entity","status":422,"detail":"owning_application doesn't exist"}

File: src/acceptance-test/java/org/zalando/nakadi/webservice/utils/TestStreamingClient.java (hunk 1 above)
Comment (v-stepanov) [inline src/acceptance-test/java/org/zalando/nakadi/webservice/utils/TestStreamingClient.java]: No, it's not possible to make it final. In the second constructor I reassign its value.

**Comment by v-stepanov** [comment]
@adyach which test fails? userJourneyM1 ?

**Comment by adyach** [comment]
@v-stepanov for me it was org.zalando.nakadi.webservice.UserJourneyAT#userJourneyHila.

File: src/acceptance-test/java/org/zalando/nakadi/webservice/utils/TestStreamingClient.java (hunk 1 above)
Comment (adyach) [inline src/acceptance-test/java/org/zalando/nakadi/webservice/utils/TestStreamingClient.java]: ups, sorry

**Comment by v-stepanov** [comment]
@adyach I fixed it now and refactored a little bit.

**Comment by adyach** [comment]
👍

**Pull request merged**